model/DeliveryContainerService.php

```diff
@@ -41,15 +41,16 @@
  */
 class DeliveryContainerService  extends ConfigurableService implements DeliveryContainerServiceInterface
 {
-
     const PROPERTY_EXCLUDED_SUBJECTS = 'http://www.tao.lu/Ontologies/TAODelivery.rdf#ExcludedSubjects';
```

Comment:
I thought that all these constants should be moved to the `model/DeliveryAssemblyService.php`

Reply:
The idea was to move the constants from constants.php file. We can have readjustments, but this is another task I think.

Reply:
I don't like an idea that we will need to use `@deprecate` in the future